axelrod/tests/strategies/test_darwin.py

```diff
@@ -47,23 +51,6 @@ def test_against_geller_and_mindreader(self):
 
         self.versus_test(axelrod.MindReader(), expected_actions=[(C, D)] * 2, attrs={'genome': [D, C]})
 
-    def test_play(self):
```

Comment:
when i refactored test_darwin, i forgot to recommend removing test_play(self).  because versus_test calls "play" these are now redundant.